src/components/layout/nav-component.vue

```diff
@@ -9,13 +9,41 @@
         <b-nav-item active href="#">Docs</b-nav-item>
         <b-nav-item active href="#">About</b-nav-item>
       </b-navbar-nav>
+      <router-link v-if="!isLogged" to="/login"
+        ><b-icon class="login__icon" icon="person-circle" height="20px"></b-icon
```

Comment:
Nadal źle rozumiesz konwencję BEM. Klasa login__icon jest niepoprawna w tym przypadku. Jako że jesteśmy w komponencie nav-component.vue to nasz BEM'owy blok powinien nazywać się 'nav'. Zatem elementy te powinny być nazwane: nav__login-icon, nav__login-text, nav__logout-btn itd.